Cargo.toml

```diff
@@ -28,6 +28,7 @@ lazy_static = "1.4.0"
 
 [dependencies.sdl3-sys]
 version = "0.4.7+SDL3-3.2.10"
+features = ["debug-impls"]
```

Comment:
Is this required? Seems like you're not using the Debug impls anywhere. I'm okay with having debug-impls since I use them quite often, just not sure if this is expected for the crate as a whole.